src/components/button-toggle-group/button-toggle-group.spec.tsx

```diff
@@ -70,6 +73,28 @@ describe("ButtonToggleGroup", () => {
     );
   });
 
+  it("when fullFlex prop is passed, renders container with flex: auto", () => {
```

Comment:
this can be a describe with to it blocks

```
describe("when fullFlex", () => {
  it("renders the container with flex: auto", () => {
    ...
    });
    
    it("renders label with width: 100%", () => {
     ...
     });
});
```